versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBStore.java

```diff
@@ -106,8 +105,14 @@ public void start() {
         final List<ColumnFamilyHandle> columnFamilyHandles = new ArrayList<>();
         transactionDB = OptimisticTransactionDB.open(dbOptions, dbPath, columnFamilies, columnFamilyHandles);
         rocksDB = transactionDB.getBaseDB();
-        valueTypeToColumnFamily = IntStream.rangeClosed(1, ValueType.values().size()).boxed().collect(
-          ImmutableMap.toImmutableMap(k -> ValueType.values().get(k - 1), columnFamilyHandles::get));
+        final ImmutableMap.Builder<ValueType<?>, ColumnFamilyHandle> builder = new ImmutableMap.Builder<>();
+        for (ColumnFamilyHandle handle : columnFamilyHandles) {
+          final String valueTypeName = new String(handle.getName(), UTF_8);
+          if (!valueTypeName.equals(new String(RocksDB.DEFAULT_COLUMN_FAMILY, UTF_8))) {
```

Comment:
The string here should be made a constant instead of recreating each time.

Reply:
Done